website/static/js/notificationsTreebeard.js

```diff
@@ -155,6 +155,7 @@ function ProjectNotifications(data) {
                 });
             }
             else if (item.parent().data.kind === 'folder' || item.parent().data.kind === 'heading' && item.data.kind === 'event') {
+                var mentionsInTitle = ~item.data.event.title.indexOf('mentions');
```

Comment:
Presumably `item` is the same object both here and inside the anonymous function assigned to `custom` below on line 174, but it's probably best to use the name as bound down there rather than up here. Let's move this line down into that function.